src/components/fields/radio-button/radio-button.tsx

```diff
@@ -98,7 +105,12 @@ export const RadioButtonGroup = (props: IGenericFieldProps<IRadioButtonGroupSche
 	const renderToggles = () => {
 		return (
 			options.length > 0 && (
-				<FlexWrapper className={className}>
+				<FlexToggleWrapper
+					className={className}
+					layoutType={
+						customOptions.styleType === "toggle" ? customOptions?.layoutType ?? "horizontal" : undefined
+					}
+				>
 					{options.map((option, index) => {
 						const radioButtonId = formatId(index);
 
```

Comment:
understand the styleType check is for type inference but it's getting repetitive, instead of that we can do a single check at the top

this same change can also be applied to `renderToggles()` in `CheckboxGroup`

```
return (
	options.length > 0 && customOptions.styleType === "toggle" && (
		<FlexToggleWrapper
			className={className}
			layoutType={customOptions?.layoutType ?? "horizontal"}
		>
			{options.map((option, index) => {
				const radioButtonId = formatId(index);

				return (
					<Toggle
						{...otherSchema}
						key={index}
						type="radio"
						id={radioButtonId}
						data-testid={TestHelper.generateId(id, "radio")}
						disabled={disabled ?? option.disabled}
						name={radioButtonId}
						indicator={customOptions?.indicator}
						styleType={customOptions?.border === false
								? "no-border"
								: "default"
						}
						checked={isRadioButtonChecked(option.value)}
						onChange={() => handleChangeOrClick(option.value)}
						error={!!error?.message}
					>
						{option.label}
					</Toggle>
				);
			})}
		</FlexToggleWrapper>
	)
);
```